doctrine.rst

```diff
@@ -598,15 +598,59 @@ the :ref:`doctrine-queries` section.
     see the web debug toolbar, install the ``profiler`` :ref:`Symfony pack <symfony-packs>`
     by running this command: ``composer require --dev symfony/profiler-pack``.
 
-Automatically Fetching Objects (ParamConverter)
------------------------------------------------
+Automatically Fetching Objects (EntityValueResolver)
+----------------------------------------------------
 
-In many cases, you can use the `SensioFrameworkExtraBundle`_ to do the query
-for you automatically! First, install the bundle in case you don't have it:
+In many cases, you can use the `EntityValueResolver`_ to do the query for you
+automatically! First, enable the feature:
```

Comment:
If I'm correct, this is now the default, right? That would mean we can remove this step here